apps/projects/templates/meinberlin_projects/project_list.html

```diff
@@ -8,12 +8,10 @@
         {% include "meinberlin_contrib/includes/filter_and_sort.html" with filter=view.filter %}
 
         <div class="l-tiles-4">
-            {% for project in project_list %}
+            {% filter_has_perm 'a4projects.view_project' request.user project_list as filtered_projects  %}
```

Comment:
unfortunately this is not possible, as custom filters only accept a single value argument [1]. But we would need the name as the permission as a string and the user as an object. 
[1] https://docs.djangoproject.com/en/1.8/howto/custom-template-tags/#writing-custom-template-filters